scripts/initOracleConfigForTokensUtils.ts

```diff
@@ -181,3 +186,49 @@ export async function initOracleConfigForTokens({ write }) {
     console.log("NOTE: executed in read-only mode, no transactions were sent");
   }
 }
+
+export async function validatePriceFeeds(tokens: Record<string, TokenConfig>) {
+  if (process.env.SKIP_PRICE_FEED_VALIDATION) {
+    console.log("skipping price feed validation");
+    return;
+  }
+
+  console.log("validating price feeds. use SKIP_PRICE_FEED_VALIDATION=true to skip");
+
+  await handleInBatches(Object.entries(tokens), 25, async (batch) => {
+    for (const [tokenSymbol, tokenConfig] of batch) {
+      await validatePriceFeed(tokenSymbol, tokenConfig);
+    }
+  });
+}
+
+async function validatePriceFeed(tokenSymbol: string, token: TokenConfig) {
+  const { priceFeed } = token;
+
+  if (!priceFeed) {
+    return;
+  }
+
+  const contract = new ethers.Contract(
+    priceFeed.address,
+    ["function decimals() view returns (uint8)", "function description() view returns (string)"],
+    ethers.provider
+  );
+  const decimals = await contract.decimals();
+  const description = await contract.description();
+  if (decimals !== priceFeed.decimals) {
+    throw new Error(`Decimals mismatch for ${tokenSymbol}: ${decimals} !== ${priceFeed.decimals}`);
+  }
+
+  const tokenSymbolReplaced =
+    {
+      "WBTC.e": "BTC",
+      tBTC: "BTC",
+      WETH: "ETH",
+      "USDC.e": "USDC",
```

Comment:
since you decided to add existing tokens, let's also add Avalanche tokens (for example, "BTC.b", "WETH.e" and "USDT.e"), could check in tokens.ts